sway/desktop/output.c

```diff
@@ -961,6 +961,11 @@ static void handle_commit(struct wl_listener *listener, void *data) {
 		wlr_damage_ring_set_bounds(&output->damage_ring, width, height);
 		wlr_output_schedule_frame(output->wlr_output);
 	}
+
+	// Next time the output is enabled, try to re-apply the gamma LUT
+	if ((event->committed & WLR_OUTPUT_STATE_ENABLED) && !output->wlr_output->enabled) {
```

Comment:
Heh, slightly confusing: We're in a post commit handler where sway output is *enabled*, but the wlr_output has been *disabled*...

But yeah, sway only changes its own state *after* committing the `wlr_output`, so it makes sense.